.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: scala
+
+scala:
+  - 2.12.9
+
+script:
+  - sbt clean compile test
```

Comment:
Is it possible to perform integration tests too?
`sbt clean compile testAll`

It will require docker.

Reply:
@AlekseiLitkovetc  added